src/profile/helpers/__tests__/updateMutationVariables.test.ts

```diff
@@ -66,113 +69,176 @@ const addresses: Address[] = [
   },
 ];
 
-describe('test getEmails function', () => {
-  test('add array is formed correctly', () => {
-    const emailObj = getEmail(emails);
-
-    expect(emailObj.addEmails.length).toEqual(1);
-  });
-
-  test('add array is null', () => {
-    const emailObj = getEmail([{ ...emails[0] }]);
-    expect(emailObj.addEmails).toEqual([null]);
-  });
-
-  test('update array is formed correctly', () => {
-    const emailObj = getEmail(emails, myProfile);
-    expect(emailObj.updateEmails).toEqual([
-      {
-        email: 'kolmas@testi.fi',
-        emailType: 'OTHER',
-        id: '234',
-        primary: false,
-      },
-    ]);
-  });
-
-  test('update array is empty (emails & profileEmails are equal)', () => {
-    const profileWithSameEmails = {
-      myProfile: {
-        ...myProfile.myProfile,
-        emails: {
-          edges: [
-            {
-              node: {
-                email: 'kolmas@testi.fi',
-                primary: false,
-                emailType: EmailType.OTHER,
-                id: '234',
-                __typename: 'EmailNode',
-              },
-              __typename: 'EmailNodeEdge',
-            },
-          ],
-          __typename: 'EmailNodeConnection',
-        },
-      },
-    } as MyProfileQuery;
-
-    const emailObj = getEmail(emails, profileWithSameEmails);
-    expect(emailObj.updateEmails).toEqual([]);
-  });
-
-  test('removeEmails field doesnt exist', () => {
-    const emailObj = getEmail(emails, myProfile);
-    expect(emailObj.removeEmails).toBeFalsy();
-  });
-
-  test('removeEmails exists', () => {
-    const emailObj = getEmail([], myProfile);
-    expect(emailObj.removeEmails).toEqual(['123', '234']);
-  });
+const phones: Phone[] = [
+  {
+    id: '123',
+    phone: '0501234567',
```

Comment:
Do we have any tests with foreign phone numbers or addresses?

Reply:
There isn't currently any validation for phone numbers and the only address based validation is that postalCode is at the most 5 characters. So at the moment every phone number will pass. But if we end up adding stricter validation, testing foreign values is a good idea!